test/fixtures_s3_test.go

```diff
@@ -0,0 +1,155 @@
+//+build integration fixtures
+
+package test_test
+
+import (
+	"fmt"
+	"github.com/applike/gosoline/pkg/blob"
+	"github.com/applike/gosoline/pkg/fixtures"
+	"github.com/applike/gosoline/pkg/mon"
+	"github.com/applike/gosoline/pkg/test/suite"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/s3/s3iface"
+	"io/ioutil"
+	"testing"
+)
+
+const (
+	basePath      = "test_data/s3_fixtures_test_data"
+	basePathPurge = "test_data/s3_fixtures_purge_test_data"
+	configName    = "test"
+)
+
+type FixturesS3Suite struct {
+	suite.Suite
+	logger     mon.Logger
+	bucketName string
+	client     s3iface.S3API
+	loader     fixtures.FixtureLoader
+}
+
+func (s *FixturesS3Suite) SetupSuite() []suite.Option {
+	return []suite.Option{
+		suite.WithConfigFile("test_configs/config.fixtures_s3.test.yml"),
+		suite.WithEnvSetup(func() error {
+			s.logger = s.Env().Logger()
+			s.bucketName = s.Env().Config().GetString("blobstore.test.bucket")
+
+			return nil
+		}),
+	}
+}
+
+func (s *FixturesS3Suite) SetupTest() error {
+	s.loader = fixtures.NewFixtureLoader(s.Env().Config(), s.logger)
+	s.client = blob.ProvideS3Client(s.Env().Config())
+
+	return nil
+}
+
+func TestFixturesS3Suite(t *testing.T) {
+	s := new(FixturesS3Suite)
+	suite.Run(t, s)
+}
+
+func (s *FixturesS3Suite) TestS3(app suite.AppUnderTest) {
+	fs := s3DisabledPurgeFixtures()
+	err := s.loader.Load(fs)
+	s.NoError(err)
+
+	input := &s3.GetObjectInput{
+		Bucket: aws.String(s.bucketName),
+		Key:    aws.String(fmt.Sprint("nyan_cat2.gif")),
+	}
+	output, err := s.client.GetObject(input)
+	s.NoError(err)
+
+	body, err := ioutil.ReadAll(output.Body)
```

Comment:
so now you are not checking the error of reading the body anymore? :see_no_evil:

Reply:
🙈 😂 Corrected.